apache/superset
# feat: hide datasource legacy based on featureflag

### SUMMARY

Hiding legacy datasource view based upon `react_crud_views` flag

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

#### Flag: False
<img width="1627" alt="Screen Shot 2020-10-22 at 8 18 34 PM" src="https://user-images.githubusercontent.com/27827808/96952644-34535280-14a4-11eb-8a79-1bc8b61b96b7.png">

#### Flag: True
<img width="1603" alt="Screen Shot 2020-10-22 at 8 20 17 PM" src="https://user-images.githubusercontent.com/27827808/96952649-361d1600-14a4-11eb-8325-6ea961baf2c6.png">

### TEST PLAN

1. Goto superset/config.py file
1. Change `ENABLE_REACT_CRUD_VIEWS` to false
1. Check if use `Use Legacy Datasource Editor` btn is hidden in `DatasourceModal` view

### ADDITIONAL INFORMATION
